src/components/Signin.jsx

```diff
@@ -0,0 +1,265 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import styled from 'styled-components';
+
+const SocialLinks = styled.ul`
```

Comment:
Prefer to create constants of styled components, in order of their appearance. That improves readability.